chatterbot/chatterbot.py

```diff
@@ -253,6 +253,11 @@ def learn_response(self, statement, previous_statement=None):
             statement.search_in_response_to = previous_statement.search_text
             previous_statement_text = previous_statement.text
```

Comment:
@gunthercox Here's the implementation of `previous_statement.text` for future use.